hs_core/templates/debug/debug_resource.html

```diff
@@ -135,4 +139,66 @@ <h2>iRODs Issues</h2>
   };
   xhttp.open("GET", "irods-issues/", true);
   xhttp.send();
+
+  // Load async data for non preferred paths in the resource
+  let xhttp_path_check = new XMLHttpRequest();
```

Comment:
I have to be honest, it's been a while since I used a vanilla `XMLHttpRequest` instead of using some js library like axios or jquery. In general, looks like we use jquery ajax in most cases in HS so that would be my default. But because this particular debug_resource case already contains other instances of "vanilla," I like what you've done sticking to that pattern.